### ISSUE TITLE
radio strips are not accessible to switch users on android

### ISSUE THREAD

**Issue by codl** [codl/forget#20]
![sscrot](https://user-images.githubusercontent.com/315139/34817053-160f3c16-f6b7-11e7-8c1f-c6d5183f0172.png)

these are what i call radio strips

they're keyboard-accessible thanks to a hack and apparently the hack is too hacky for android's switch support cos it just passes over them completely

theyre still accessible through the crosshair thing but that's messy

**Comment by codl** [comment]
![screenshot_20180111-102543](https://user-images.githubusercontent.com/315139/34818042-f77e9c76-f6b9-11e7-907a-e74104234f2d.jpg)

**Comment by codl** [comment]
More android accessibility:

"Select to speak" does not read them at all

Talkback reads them but scrolls to the top of the page while doing so

The problem might be that the radio buttons are hidden in the top left of the page

**Comment by codl** [comment]
no. begone
